__tests__/app.test.js

```diff
@@ -237,4 +237,35 @@ describe("POST /api/articles/:article_id/comments", () => {
         expect(response.body).toEqual({ msg: "Invalid Input!" });
       });
   });
+  it('should return a 404 if given a username that doesnt exist', () => {
+    const newComment = { username: "BillyBob", body: "cba to register" };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(404)
+      .then((response) => {
+        expect(response.body).toEqual({ msg: "Input not found!" });
+      });
+  });
+  it('should ignore additional properties in the comment', () => {
+    const newComment = { username: "icellusedkars", body: "cba to register", votes: 1};
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(400)
+      .then((response) => {
+        expect(response.body).toEqual({ err: "Invalid inputs provided!" });
+      });
```

Comment:
When August mentioned ignoring additional keys, we would still consider this a happy path. The extra keys will be ignored and a 201 status will be provided as normal. The user has done the _right_ thing, just with some additional _wrong_ things which can simply be ignored, so no error 👍